### DISCUSSION TITLE
astronomy photo of the day (apod)

### DISCUSSION THREAD

**Discussion by fauwara** [lennart-rth/Live-Earth-Wallpapers#20] [Show and tell]
worked on apod issue (web scraped). have to work on resize so that image does not get squashed and merge with new formatted code cause my code is in the old version oops.
![image](https://user-images.githubusercontent.com/53698026/201528154-94ae716e-c36c-4c7f-94df-f4a2a9ab2af9.png)

also why did you suggest not to use api?

**Comment by lennart-rth** [comment]
As I understand it, the api is not designed for long-term use. Its just a demo to test the functionality. It would be not very fair to use this and therefore possibly direct a lot of traffic to it.
But I could be wrong! Haven´t had a detailed look at it.
